quadriga/pom.xml

```diff
@@ -186,13 +186,14 @@
             <artifactId>mongodb-driver-sync</artifactId>
             <version>${mongo-version}</version>
         </dependency>
-
+        
         <!-- https://mvnrepository.com/artifact/org.neo4j/neo4j-ogm-http-driver -->
-        <dependency>
-            <groupId>org.neo4j</groupId>
-            <artifactId>neo4j-ogm-http-driver</artifactId>
-            <version>3.2.11</version>
-        </dependency>
+		<dependency>
+		    <groupId>org.neo4j</groupId>
+		    <artifactId>neo4j-ogm-http-driver</artifactId>
+		    <version>3.3.3</version>
+		</dependency> 
```

Comment:
indentation

Reply:
The indentation seems right, also there wasn't any code factor issue.

Reply:
It's definitely more indented than the rest. And code factor doesn't check the indentation of XML files most likely